src/fhir/api.test.ts

```diff
@@ -14,20 +87,12 @@ describe("FHIR resources", () => {
    * Then the fhir library should pass validation of the bundle
    */
   test('Bundle is valid', () => {
-    const patientForm = {
-      mrn: "969977",
-      firstName: "Donald",
-      lastName: "Duck",
-      dateOfBirth: "2012-03-04",
-      gender: Patient.GenderEnum.Male,
-      familyNumber: "Z409929",
-    };
 
-    const bundle = createBundle(patientForm);
+    const bundle = createBundle(testValues);
 
     const output = fhir.validate(bundle);
-    console.info("Validation output")
-    console.info(JSON.stringify(output.messages))
+    console.info("Validation output");
+    console.info(JSON.stringify(output.messages, null, 2));
```

Comment:
So ideally I'd like to have them print upon a test failure, because then it'd be useful for debugging but I couldn't see a nice way to do that. Is there something I'm missing other than doing that manually?